templates/windows/chocolateyUninstall.ps1.erb

```diff
@@ -0,0 +1,20 @@
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$contentFile=(join-path -path "$toolsDir" -childpath "file-list") + ".txt"
+if (Test-Path -path "$contentFile") {
+  Write-Debug "Removing files defiend in `'$contentFile`'"
+  $contentFile
+  $contents=get-content "$contentFile"
```

Comment:
What is the encoding of `file-list.txt`? You may need to tell PowerShell what the encoding is if it is not UTF-8 or ASCII.